cilium-cli/utils/features/features.go

```diff
@@ -375,6 +377,10 @@ func (fs Set) ExtractFromConfigMap(cm *v1.ConfigMap) {
 		Enabled: cm.Data[string(ClusterMeshEnableEndpointSync)] == "true",
 	}
 
+	fs[PolicyDefaultLocalCLuster] = Status{
+		Enabled: cm.Data[string(PolicyDefaultLocalCLuster)] == "true",
+	}
```

Comment:
This assumes that both clusters are configured in the same way, but that's a common assumption for most of the other features as well, so it seems reasonable to me at the moment. We can always relax this requirement in a second moment.